src/components/profile.tsx

```diff
@@ -7,11 +7,25 @@ interface ProfileProps {
   session: Session;
 }
 
+function getInitials(name: string) {
+  const firstName = name.split(" ")[0];
+  const lastName = name.split(" ")[1];
+
+  return `${firstName?.at(0)}${lastName?.at(0)}`;
+}
+
 export default function Profile({ session }: ProfileProps) {
   return (
     <Avatar>
-      <AvatarImage className="rounded-full" src={session.user.image ?? ""} alt="profile picture" />
-      <AvatarFallback>CN</AvatarFallback>
+      <AvatarImage
+        // eslint-disable-next-line
```

Comment:
a classic

Reply:
The Avatar image ha a mechanism where it will use `AvatarFallback` if the image is not found. This doesn't play nice with typescript since the image can be `undefined` and typescript has no way of knowing that this is actually fine since the `non-null-assertion`. 

We are using strict `eslint` which doesn't allow such things since the ignore.